docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst

```diff
@@ -193,3 +193,19 @@ the GPUs used for the experiment to be evenly distributed among the compute node
 
    For more information on MPS, refer to the `NVIDIA Multi-Process Service (MPS) Documentation
    <https://docs.nvidia.com/deploy/mps>`__.
+
+-  Experiments on CPU-only clusters will fail when the requested slot count exceeds the maximum 
+   number of CPUs on any single node. This behavior is due to a limitation of the Slurm workload 
```

Comment:
i wouldn't necessarily call this a limitation of slurm, but a limitation of how determined launches on slurm?

Reply:
From a user perspective, I would say this is a limitation of Slurm, as users expect an abstraction from slurm when requesting N CPUs. Ideally, slurm should be able to derive the node allocation based on the given CPU request and distribute the work across multiple nodes. However, slurm does not do that now.